src/silx/image/marchingsquares/_mergeimpl.pyx

```diff
@@ -54,7 +54,7 @@ cimport cython
 
 from libc.stdint cimport int8_t, uint8_t, int32_t, uint32_t
 
-from ...utils._have_openmp cimport COMPILED_WITH_OPENMP
+from ._have_openmp cimport COMPILED_WITH_OPENMP
```

Comment:
`_have_openmp` moved here since it is the only file using it so far and this makes `silx.utils` a pure Python sub-package